crates/sdk/src/migrations.rs

```diff
@@ -564,7 +544,6 @@ impl TypeHash
         typehash!(SerializeWrapper<BTreeMap<String, Address>>);
 }
 
-#[cfg(feature = "migrations")]
```

Comment:
the sdk always has migrations enabled by default?

Reply:
it's still guarded but I just moved it onto the `mod migrations` instead